Qt data hub removal (#6733)
mendix/docs: content/en/docs/refguide/modeling/domain-model/associations/association-properties.md

#### 2.5.1 One-Way Navigable Associations {#one-way-navigable}

> One-way navigable associations are associations that allow navigation to associated entities only in a single direction: from parent to child. These associations are typically introduced by OData service integration, either through an external provider or the Catalog, in [external entities](/refguide/external-entities/#associations). The major implications of one-way navigability are related to XPath use cases such as constraints and queries.

Reviewer: I think we should remove the subclause about external provider and data hub. And just talk about external entities (in the interests of making it more generic). But perhaps we need to check this with someone (Maurits or Joost?). Might as well make it more generic by removing OData as well. So perhaps:
```suggestion
One-way navigable associations are associations that allow navigation to associated entities only in a single direction: from parent to child. These associations are typically introduced by integration using [external entities](/refguide/external-entities/#associations). The major implications of one-way navigability are related to XPath use cases such as constraints and queries.
```
